dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkAspect.java

```diff
@@ -36,30 +36,20 @@
  * </pre>
  */
 public class UnitOfWorkAspect {
+    private static final ThreadLocal<UnitOfWorkContext> CONTEXT = new ThreadLocal<>();
```

Comment:
Another option here would be to create multiple ThreadLocals, one for each piece of state.